simpleflow/swf/task.py

```diff
@@ -168,6 +168,10 @@ def payload(self):
 
     @property
     def task_list(self):
+        get_task_list = getattr(self.workflow, 'get_task_list', None)
+        if get_task_list:
+            return get_task_list(self.workflow, *self.args, **self.kwargs)
```

Comment:
Si je fais un truc comme ca:
```python
custom_task_list = get_task_list(self.workflow, *self.args, **self.kwargs)
if custom_task_list:
    return custom_task_list
```

Ca permet de fallback sur le task_list par défault dans le cas où il est pas précisé non ?

Reply:
Nevermind, I changed it on the workflow declaration.